src/Grids/orthogonal_spherical_shell_grid.jl

```diff
@@ -611,17 +611,17 @@ OrthogonalSphericalShellGrid(FT::DataType; kwargs...) = OrthogonalSphericalShell
 
 function load_and_offset_cubed_sphere_data(file, FT, arch, field_name, loc, topo, N, H)
 
```

Comment:
```suggestion
loc = map(instantiate, loc)
```

where

```
instantiate(T::DataType) = T()
instantiate(t) = t
```

I've been ising the convention that "uninstantiated" locations are `Loc` while instantiated are `loc`, following the distinction between types and instances